src/components/dashboard/announcements.tsx

```diff
@@ -0,0 +1,21 @@
+// Components
+import Title from "@components/global/title";
+import Icon from "@components/global/icon";
```

Comment:
I would personally name this component `MaterialIcon` to avoid confusing with other icon library